frontend/src/v5/ui/components/shared/sso/microsoftText.component.tsx

```diff
@@ -59,7 +59,7 @@ export const MicrosoftText = ({ title, className }: MicrosoftTextProps) => (
 				`}
 				values={{
 					TermsLink: (label) => <Link to={TERMS_ROUTE} target="_blank">{label}</Link>,
-					PrivacyLink: (label) => <Link to={PRIVACY_ROUTE} target="_blank">{label}</Link>,
+					PrivacyLink: (label) => <a href={PRIVACY_ROUTE} target="_blank">{label}</a>,
```

Comment:
The link is rendered differently, can we unify it?

on staging
![image](https://github.com/3drepo/3drepo.io/assets/11945337/3a979801-3bf6-4869-bc22-34063bb63cdb)
on this branch
![image](https://github.com/3drepo/3drepo.io/assets/11945337/544879e1-4b5d-481b-91c6-9bab6356006a)